trunk/source/IcfgBuilder/LicenseTemplate.md

```diff
@@ -0,0 +1,23 @@
+Copyright (C) @{daterange} @{author:r}
+
+This file is part of the ULTIMATE RCFGBuilder plug-in.
```

Comment:
Shouldn't the objective be called ICFGBuilder instead of RCFGBuilder in the whole license template file?